plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/TestSqlServerConnectorTest.java

```diff
@@ -277,27 +278,55 @@ private List<String> testTableNameTestData()
                 .build();
     }
 
-    @Test
+    @RepeatedTest(100)
```

Comment:
I understand this is only for development, and not meant for final merge?

Reply:
Right, I just dropped the tmp commit.